datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneDataMapFactoryBase.java

```diff
@@ -61,6 +61,9 @@
 @InterfaceAudience.Internal
 abstract class LuceneDataMapFactoryBase<T extends DataMap> extends DataMapFactory<T> {
 
+  static final String FLUSH_CACHE = "flush_cache";
```

Comment:
Can you explain the intention of these two properties just like you wrote in PR description.

Reply:
besides, what's the unit of this value? Better to add description for it.